lib/theme_check/checks/missing_template.rb

```diff
@@ -28,7 +28,11 @@ def on_section(node)
     private
 
     def ignore?(path)
-      @ignore_missing.any? { |pattern| File.fnmatch?(pattern, path) }
+      all_ignored_patterns.any? { |pattern| File.fnmatch?(pattern, path) }
+    end
+
+    def all_ignored_patterns
+      @all_ignored_patterns ||= @ignore_missing + (ignored_patterns || [])
```

Comment:
This was also my first implementation 😅 

It is for the unit tests. 

The Config class sets up the ignored_patterns but the Check class doesn't have that. So when we unit test with MissingTemplate.new, since we're using `attr_accessor` for ignored_patterns, we get nil. And so I added that there (perhaps by being lazy and without wanting to change the attr_accessor in Check.rb 😅.

Reply:
But you're right that it's probably best if I do it there instead of here.